python/tank/authentication/shotgun_shared/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+"""
```

Comment:
I actually specifically suggested to move the files into a subfolder. I want it to be easy to tell if this is code from us or code shared between RV and Toolkit.